docs/_includes/attr-doc.adoc

```diff
@@ -65,16 +68,27 @@ You can also set the base path to images (default: _empty_), icons (default: `./
 
 When you find yourself typing the same text repeatedly, or text that often needs to be updated, consider assigning it to a document attribute and use that instead.
 
+NOTE: If you're used to XML, you'll know document attributes as User Entities.
```

Comment:
Thanks for all this feedback. Some of the comments related to the original
content that I simply reordered. Which is a prime example of surfacing
errors by improving readability.

I'll work on incorporating this feedback during my evening and we should be
good to go.

On Wed, 27 Jan 2016 at 08:15 Dan Allen <notifications@github.com> wrote:

> In docs/_includes/attr-doc.adoc
> <https://github.com/asciidoctor/asciidoctor.org/pull/484#discussion_r50909837>
> :
>
> > @@ -65,16 +68,27 @@ You can also set the base path to images (default: _empty_), icons (default: `./
> >
> >  When you find yourself typing the same text repeatedly, or text that often needs to be updated, consider assigning it to a document attribute and use that instead.
> >
> > +NOTE: If you're used to XML, you'll know document attributes as User Entities.
>
> Perhaps write this as:
>
> If you're familiar with writing in XML, you'll recognize document
> attributes as (custom text) entities.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/asciidoctor/asciidoctor.org/pull/484/files#r50909837>.
>
-- 

Sent from Mobile.